core/src/main/java/pl/ds/howlite/components/models/TitleComponent.java

```diff
@@ -27,7 +27,7 @@
 import pl.ds.howlite.components.Grid;
 import pl.ds.howlite.components.Styled;
 
-@Model(adaptables = Resource.class)
+@Model(adaptables = Resource.class, defaultInjectionStrategy = OPTIONAL)
```

Comment:
I think that
import static org.apache.sling.models.annotations.DefaultInjectionStrategy.OPTIONAL;
is missing :)

Reply:
Sorry about that :) added